static/js/ranking/controllers/ranking.controller.js

```diff
@@ -6,36 +6,75 @@
     .module('crowdsource.ranking.controllers')
     .controller('RankingController', RankingController);
 
-  RankingController.$inject = ['$scope', '$log', '$mdToast', '$http', 'RankingService'];
```

Comment:
I think we should keep the turkopticon rating example. It has helped new folks getting started with our framework. (may be rename it to something else for now)